### PR TITLE
Developement

### PR DESCRIPTION
**Pull request by farcry**
merging development with master

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,362 @@
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+  <meta charset="utf-8">
+  <title></title>
```
Comment (JamesOkunlade) [inline index.html]: Element title must not be empty.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+      		<span>A</span>
+      	</button>
+
+  	</div>
+    <!-- end of options container -->
+  </header>
+  <div class="container">
+    <main>
+      
+        <iframe width="853" height="480" src="https://www.youtube.com/embed/V74l_zS1x8E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
```
Comment (JamesOkunlade) [inline index.html]: The frameborder attribute on the iframe element is obsolete. Use CSS style instead.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,308 @@
+*{
+    box-sizing: border-box;
```
Comment (JamesOkunlade) [inline css/style.css]: The box-sizing property isn't supported in IE6 and IE7. Research and use vendor prefixes when declaring border-box.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+	width: 50%;
+}
+.stats-left{
+	line-height: 54px;
+}
+.stats-list{
+	position: relative;
+	list-style-type: none;
+	display: flex;
+	/*flex-direction: row-reverse;*/
```
Comment (JamesOkunlade) [inline css/style.css]: Remove styling comment from code. Not needed.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+	color: #909090;
+}
+.stats-list li{
+	display: flex;
+}
+.stats-list li span{
+	padding-left: 8px;
+	font-size: 13px;
+	line-height: 20px;
+	/*padding-top: 2px;*/
```
Comment (JamesOkunlade) [inline css/style.css]: Remove this line as well. Always leave your CSS code free of unneeded styling comments.

**Comment by farcry** [comment]
Great we will do it tomorrow and come back to you in the next project

**Pull request merged**